torch/_meta_registrations.py

```diff
@@ -3015,6 +3030,15 @@ def meta__foreach_unaop(self):
     return [torch.empty_like(s) for s in self]
 
 
+@register_meta_foreach({aten._foreach_abs.default: aten.abs.default})
```

Comment:
Is there a easier way to figure out `aten.abs.default` from `aten._foreach_abs.default` ?

Reply:
just do some string wrangling on the names, no?